### PR TITLE
PWN-9272, PWN-9274 - Striga. Off-ramp cell + withdraw flow + otp

### PR DESCRIPTION
**Pull request by eduardmaximovich**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9272
https://p2pvalidator.atlassian.net/browse/PWN-9274

## Description of Work

- Striga off ramp cell
- Striga off ramp sms OTP
- microfixes

### CODE REVIEW

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletViewState.kt [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package org.p2p.wallet.home.ui.wallet
 
 import org.p2p.wallet.home.ui.wallet.mapper.model.StrigaKycStatusBanner
+import org.p2p.wallet.striga.offramp.models.StrigaOffRampToken
 import org.p2p.wallet.striga.onramp.interactor.StrigaOnRampToken
 
 data class WalletViewState(
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletViewState.kt:7]: WalletStrigaViewState :))

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/handlers/StrigaHandler.kt [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import org.p2p.wallet.infrastructure.network.provider.TokenKeyProvider
+import org.p2p.wallet.striga.offramp.withdraw.interactor.StrigaWithdrawInteractor
+import org.p2p.wallet.striga.onramp.interactor.StrigaOnRampInteractor
+import org.p2p.wallet.striga.sms.interactor.StrigaOtpConfirmInteractor
+import org.p2p.wallet.striga.user.interactor.StrigaUserInteractor
+import org.p2p.wallet.striga.wallet.interactor.StrigaNoBankingDetailsProvided
+import org.p2p.wallet.striga.wallet.interactor.StrigaWalletInteractor
+import org.p2p.wallet.transaction.model.HistoryTransactionStatus
+
+class StrigaHandler(
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/handlers/StrigaHandler.kt:29]: WalletStrigaHandler

**Review by gslevinkov — APPROVED** [review]

**Comment by gslevinkov** [comment]
so good! thanks

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletViewState.kt (hunk 1 above)
Comment (eduardmaximovich) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletViewState.kt:7]: We definitely should rename "Wallet" screen to "Striga"

**Pull request merged**